cpp/src/Ice/LocatorInfo.cpp

```diff
@@ -38,9 +38,8 @@ namespace
             {
                 LocatorInfo::RequestPtr request = shared_from_this();
                 _locatorInfo->getLocator()->findObjectByIdAsync(
-                    _reference->getIdentity(),
-                    [request](const optional<ObjectPrx>& object) { request->response(object); },
-                    [request](exception_ptr e) { request->exception(e); });
+                    _reference->getIdentity(), [request](const optional<ObjectPrx>& object)
```

Comment:
Why this change?

Reply:
@bernardnormier was running version 17 when he ran it yesterday. I've updated to 18.

Reply:
Mmm, I think the previous style was clear maybe a default value change?

Reply:
Maybe, do they have a args per line setting maybe

Reply:
Seems like this should be handled by `BinPackArguments`

Reply:
When I manually run format on save in vsocde it undoes this change. Let me look a little closer.

Reply:
Right, and we set it to false, so not clear why it changed.

Reply:
ok, I've added `BinPackArguments = false`

Reply:
I prefer this updated style.